server/odc-service/src/main/java/com/oceanbase/odc/service/connection/model/ConnectionConfig.java

```diff
@@ -302,6 +302,9 @@ public class ConnectionConfig
     @JsonIgnore
     private OBInstanceType instanceType;
 
+    @JsonIgnore
+    private OBInstanceRoleType instanceRoleType;
```

Comment:
It's too customized. This class also needs to be oriented to other data sources. This way it will be too closely bound to OB.

Reply:
I don't think it's called 'customized'. Yes, the class `ConnectionConfig` is designed to be oriented to different types of data sources, but it doesn't mean we should only involve the common part of each data source. Think about the design of `DBTable` in schema accessors.